scripts/reverseproxy-facts.py

```diff
@@ -43,6 +44,10 @@ def generateFacts(original_facts, reverse_proxy_host):
       failed_names['description'].append(host)
       continue
 
+    # Skip hosts to ignore
+    if host in facts['ignore_hosts']:
```

Comment:
It seems like there would be an error if there is no key `ignore_hosts`, yet the option is not declared mandatory in readme. How about `if 'ignore_hosts' in facts and host in facts['ignore_hosts']:`?